cvmfs/shrinkwrap/fs_traversal.cc

```diff
@@ -564,6 +578,14 @@ bool SyncFull(
       return false;
     }
 
+    if (stat_update_period_ > 0 &&
+        time(NULL)-last_print_time > stat_update_period_) {
```

Comment:
You'll find a `platform_monotonic_time()` in platform_linux.h and platform_osx.h, which is more appropriate here.